lib/glimesh_web/templates/oauth2_provider/authorized_application/index.html.eex

```diff
@@ -0,0 +1,25 @@
+<div class="container" style="text-align:center;">
+  <h1><%= gettext("Authorized Applications") %></h1>
+
+  <table style="width:50%; margin: 0 25%">
```

Comment:
```suggestion
  <table class="table">
```

Reply:
this should not be a full width table which is why its set to 50%